androidApp/src/main/java/com/example/tasky/android/theme/MyTypography.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.tasky.android.theme
+
+import androidx.compose.material3.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.googlefonts.Font
+import androidx.compose.ui.text.googlefonts.GoogleFont
+import androidx.compose.ui.unit.sp
+import com.example.tasky.android.R
+
+val provider =
+    GoogleFont.Provider(
+        providerAuthority = "com.google.android.gms.fonts",
+        providerPackage = "com.google.android.gms",
+        certificates = R.array.com_google_android_gms_fonts_certs,
+    )
```

Comment:
There is nothing wrong with this approach 🙂. I want to highlight an alternative that has the advantage of the fonts being available regardless of the device's internet connection. We provided you with the `.ttf` font assets, which you can download (see below for an explanation). Once you have them, you can add them to your `font` package and then use them in your project.

<img width="205" alt="Screenshot 2024-09-27 at 07 26 38" src="https://github.com/user-attachments/assets/697d1438-c253-481c-8e78-a777c935b913">

You can then use them like this:

```
val Typography = Typography(
    bodySmall = TextStyle(
        fontFamily = Inter,
        fontWeight = FontWeight.Normal,
        fontSize = 12.sp,
        lineHeight = 20.sp,
        color = TaskyBlack
    ),
    // …
)
```

The resources are located [here](https://pl-coding.mymemberspot.de/library/tTq0xvOnQ6d7DgrGSKWM/JD81zm0h1p6w4izD5s0l/SpJxcWLXQYhINOoawOZu/details) in the **DROP TABLE Video Portal**’s Introduction section, the **Tasky Resources** item.